src/components/footer/footer.js

```diff
@@ -9,13 +9,14 @@ const Footer = props => {
           <div className="footerLeft">
             <h2>SITEMAP</h2>
             <ul>
-            <li><a href={`https://${window.BASE_URL}/data`}> Explore Data</a > </li>
+            <li><a href={`https://${window.BASE_URL}/about/about-smart-columbus`}>About Us</a> </li> 
+            <li><a href={`https://${window.BASE_URL}/data`}> Explore Data</a > </li> 
             <li><a href={`https://d2rfd3nxvhnf29.cloudfront.net/2019-09/SCC-D-Data%20Privacy%20Plan-FINAL-20190906%5B1%5D_0.pdf`}> Data Privacy Plan</a > </li>
-            <li><a href={`https://${window.BASE_URL}/about/about-smart-columbus`}>About Us</a> </li>
+            <li><a href={`https://${window.BASE_URL}/about/the-power-of-data`}>The Power of Data</a> </li>
```

Comment:
Since this PR is already here, can you update any link that points to joomla to include the wwww - www.${window.BASE_URL} - they are like this in the header already.  It saves one redirect for the user.